dupe-detection/main.go

```diff
@@ -544,21 +568,31 @@ func compute_parallel_bootstrapped_kendalls_tau_func(x []float64, list_of_finger
 	robust_average_tau := make([]float64, len(list_of_fingerprints_requiring_further_testing_2))
 	robust_stdev_tau := make([]float64, len(list_of_fingerprints_requiring_further_testing_2))
 
-	for fingerprintIdx, y := range list_of_fingerprints_requiring_further_testing_2 {
-		list_of_bootstrap_sample_indices := make([][]int, number_of_bootstraps)
-		x_bootstraps := make([][]float64, number_of_bootstraps)
-		y_bootstraps := make([][]float64, number_of_bootstraps)
-		bootstrapped_kendalltau_results := make([]float64, number_of_bootstraps)
-		for i := 0; i < number_of_bootstraps; i++ {
-			list_of_bootstrap_sample_indices[i] = randInts(0, original_length_of_input-1, sample_size)
-		}
-		for i, current_bootstrap_indices := range list_of_bootstrap_sample_indices {
-			x_bootstraps[i] = arrayValuesFromIndexes(x, current_bootstrap_indices)
-			y_bootstraps[i] = arrayValuesFromIndexes(y, current_bootstrap_indices)
+	g, _ := errgroup.WithContext(context.Background())
+	for i, y := range list_of_fingerprints_requiring_further_testing_2 {
```

Comment:
@maxhora you need to start getting rid of from `_` (underscores), I am going to turn on CircleCi checks for your packages too shortly, and there will be a looot of "swearing"

Reply:
@levkoburburas oh noo :) got it, thanks